src/main/java/com/minecolonies/colony/permissions/Permissions.java

```diff
@@ -226,11 +231,31 @@ public void loadPermissions(@NotNull NBTTagCompound compound)
             }
             catch (IllegalArgumentException e)
             {
-                /**
+                /*
                  * Intentionally left empty. Happens when the UUID hasn't been saved yet.
                  */
             }
         }
+
+        final Map.Entry<UUID, Player> owner = getOwnerEntry();
+        if(owner == null && ownerUUID != null)
+        {
+            GameProfile player = FMLCommonHandler.instance().getMinecraftServerInstance().getPlayerProfileCache().getProfileByUUID(ownerUUID);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'player' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)